src/v3/src/transformer/profile/transformEnrollProfile.ts

```diff
@@ -67,7 +72,7 @@ export const transformEnrollProfile: IdxStepTransformer = ({ transaction, formBa
       ...passwordElement.options,
       attributes: {
         ...passwordElement.options?.attributes,
-        autocomplete: 'new-password',
+        autocomplete: disableAutocomplete ? 'off' : 'new-password',
```

Comment:
Should we consider a separate transformer that overwrites this for us based on `disableAutocomplete`? Thinking of something like the LTR transformer @leemchale-okta just added

Reply:
Oh yes this is a good idea 🙏